### PR TITLE
Adds an apache2 alpine image

### PR DESCRIPTION
**Pull request by kimpepper**
Front-end developers just need an apache instance to serve static files (e.g. for a generated styleguide). 

This adds an apache image with skpr tooling, as well as a dev image which includes front-end dev tools such as node, npm and yarn.

### CODE REVIEW

File: apache2/alpine/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  bash \
+  bash-completion \
+  curl \
+  make \
+  rsync \
+  unzip \
+  vim
+
+# Tuner - https://github.com/previousnext/tuner
+RUN curl -sL https://github.com/previousnext/tuner/releases/download/1.0.0/tuner-linux-amd64 -o /usr/local/bin/tuner && \
```
Comment (nicksantamaria) [inline apache2/alpine/Dockerfile]: I reckon tuner can be ditched for this image.

File: apache2/alpine/Makefile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+IMAGE=previousnext/apache2-alpine
+VERSION=1.x
+ALPINE_VERSION=3.9
+
+# Allows for providing unique tags for images based on when they were built.
+# Ideal for debugging!
+TIMESTAMP=$(shell date +%F)
+
+define push
```
Comment (nicksantamaria) [inline apache2/alpine/Makefile]: ditch this `push` target.

File: apache2/alpine/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+# https://www.camptocamp.com/en/actualite/flexible-docker-entrypoints-scripts/
+COPY docker-entrypoint.sh /docker-entrypoint.sh
+RUN chmod +x /docker-entrypoint.sh
+COPY /docker-entrypoint.d/* /docker-entrypoint.d/
+
+COPY entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
+
+ENTRYPOINT ["/docker-entrypoint.sh"]
+# @todo, Change to use "httpd -D FOREGROUND" once entrypoint.sh can be retired.
```
Comment (nicksantamaria) [inline apache2/alpine/Dockerfile]: Can we implement this `@todo`?

**Comment by kimpepper** [comment]
Discussed an alternative approach, so won't be needing this after all.

**Pull request closed without merging**